applications/root.md

```diff
@@ -0,0 +1,366 @@
+
+In Score Voting, each voter assigns a numerical score to each proposal, and the scores are tallied up to determine the winner. The proposal with the highest overall score is typically chosen as the winner. This system allows voters to express a more granular preference for each proposal, and it can be more effective at identifying the most widely supported proposal, even in cases where there is no clear consensus among voters.
+
+Score Voting can also be modified to require that each voter allocate a total of 100 points across all the proposals, to ensure that each voter is submitting a complete preference ranking. This is sometimes called "100-point voting" or **"range normalization"**.
+
+[Warren D. Smith](https://www.rangevoting.org/WarrenSmithPages/homepage/rangevote.pdf) and Jan Kok both independently proposed the idea of score voting in the early 2000s, and since then it has been the subject of much research and experimentation in the field of electoral systems.
+
+### Overview
+
+Root is born from experience gained in ongoing applied contribution to on-chain governed networks such as Edgeware and aims to address many of the current limitations of existing approaches facing Substrate based chains, when it comes to existing issues around voting engagement, ecosystem talent development and the formation of effective, efficient and reputable on-chain organisations.  
+
+By returning to first principles we aim to standardize proposal making processes for the Substrate ecosystem, optimising for _intelligent adoption_ rather than simply gamifying governance in a way that may create unintended and undesirable outcomes over the long term. 
+
+Root is built around a collective creation and decision making system that uses a composable voting system inspired by Borda's Count and ranged normalization to compute voting and allow contributors to create _preferendums_ drectly linked to governance pallets.
+
+Our goal is to facilitate low level consensuses for proposals from initiating teams, before they can smoothly send their proposal to gov1/2 from/to any Substrate network via root.origin and inspired by [shokunin style](https://polkaverse.com/@shokunin/let-s-use-pallet-proxy-for-chaos-and-good-32567) optimistic pure proxy organisations in the process.
+
+Teams can now benefit from an environment where they can collaborate and reach consensus to get supported from the ground up before they send their proposals to a binary Y/N referendum.
+
+We built this [ggdoc hack](https://docs.google.com/document/d/1-g2A387hsOcWc7_UJWwwtgr2gyE553gvFURTVXOEErE/edit?usp=sharing) to demonstrate what we would like to achieve. Check out the comments to undestand the logic.
```

Comment:
The doc contains this text: "Engagement can have a [...] financial and creative upside". Who provides this financial upside? Contributing to how your treasury spends funds is already a financial upside for most.

Reply:
Some areas of research we already observed: treasuries create an adversarial relationship between holders and R&D teams as holders and especially speculators often see treasuries as something that needs to be protected and kept away from builders who are seen as free riders. This honeypot issue is something we're aiming to solve, kabocha not having any treasury in its runtime.